### ISSUE TITLE
Calling easeTo with empty CameraOptions crashes

### ISSUE THREAD

**Issue by jliumo** [mapbox/mapbox-maps-android#2454]
## Environment

- Android OS version: 13, probably all
- Devices affected: All tested
- Maps SDK Version: 11.5.1, 11.6.0-rc.1

## Observed behavior and steps to reproduce
Calling com.mapbox.maps.plugin.animation.easeTo with empty CameraOptions crashes the application, because it tries to spread an empty array.
```
java.lang.ArrayIndexOutOfBoundsException: length=0; index=0
   at android.animation.AnimatorSet.playTogether(AnimatorSet.java:233)
   at com.mapbox.maps.plugin.animation.CameraAnimationsPluginImpl.startHighLevelAnimation(CameraAnimationsPluginImpl.kt:1005)
   at com.mapbox.maps.plugin.animation.CameraAnimationsPluginImpl.easeTo(CameraAnimationsPluginImpl.kt:723)
   at com.mapbox.maps.plugin.animation.CameraAnimationsUtils$easeTo$1.invoke(CameraAnimationsExt.kt:40)
   at com.mapbox.maps.plugin.animation.CameraAnimationsUtils$easeTo$1.invoke(CameraAnimationsExt.kt:40)
   at com.mapbox.maps.MapboxMap.cameraAnimationsPlugin(MapboxMap.kt:2245)
   at com.mapbox.maps.plugin.animation.CameraAnimationsUtils.easeTo(CameraAnimationsExt.kt:40)
   at com.mapbox.maps.plugin.animation.CameraAnimationsUtils.easeTo$default(CameraAnimationsExt.kt:36)
```
## Expected behavior
The call shouldn't do anything.
## Notes / preliminary analysis
This causes problems for us, because even asynchronous cameraForCoordinates called during layout changes sometimes returns empty cameraState causing whole application to crash, while it can be checked by com.mapbox.maps.util.isEmpty I think it would be more developer friendly if it didn't cause crash
```
W  [maps-core]: {}[General]: Unable to calculate camera for given bounds/geometry, padding is greater than map's width or height.
E  [maps-android\Mbgl-MapboxMap]: Error occurred in asynchronous cameraForCoordinates: Unable to calculate cameraForCoordinates, either padding exceeds screensize or screensize is not set, empty cameraState will be returned
```

**Comment by flasher297** [comment]
Thank you for the detailed description of the issue.
We've raised an internal ticket to fix it and will come back as soon as it is ready.

**Comment by flasher297** [comment]
@jliumo we introduced a fix for this bug in [v11.7.0-rc.1](https://github.com/mapbox/mapbox-maps-android/releases/tag/v11.7.0-rc.1) release of SDK.

> Fix a crash when calling CameraAnimationsPlugin.easeTo() with empty camera options or CameraAnimationsPlugin.playAnimatorsSequentially() / CameraAnimationsPlugin.playAnimatorsTogether() with an empty array of animators.
> Fix ongoing animations being canceled when CameraAnimationsPlugin.flyTo() with empty camera options is called.

Test it and rise a ticket if you still have an issue.
